assignments/objects.js

```diff
@@ -18,20 +18,60 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const intern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+  speak: function() {
+    console.log("Hello, my name is " + this.name);
+  }
+}
+const intern3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const intern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.comu",
+  "gender": "F"
+  multiplyNums: function(a,b) {
+    return a * b;
+  }
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+intern1["name"];
 
 // Kennan's ID
+intern2["id"];
 
 // Keven's email
+intern3["email"];
 
 // Gannie's name
+intern4["name"];
 
 // Antonietta's Gender
+intern5["gender"];
 
 // ==== Challenge 3: Object Methods ==== 
```

Comment:
You have the `speak` and `multiplyNums` on your objects already. Though, you should've console logged here to show they work. You could also have shown how to add a method to an object after the object declaration.